examples/run_over_exo.sh

```
@@ -0,0 +1,68 @@
+        result=$(curl -sf -X POST http://localhost:52415/v1/chat/completions \
+            -H "Content-Type: application/json" \
+            -d "{\"model\":\"${exo_model}\",\"messages\":[{\"role\":\"user\",\"content\":\"hi\"}],\"max_tokens\":1,\"stream\":false}" 2>/dev/null)
+        if [[ "$result" != *"No instance found"* ]] && [[ "$result" != *"404"* ]] && [[ -n "$result" ]]; then
+            echo " ready."
+            break
+        fi  
+        echo -n "." 
+        sleep 2
+    done
+    echo ""
+    source "$HOME/Developer/mlx-omni-server/.venv/bin/activate"
```

Shellcheck (0.11.0)

[info] Not following: ./Developer/mlx-omni-server/.venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)